src/components/AdminDashboard/RejectPendingApplication/index.tsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+
+import { Clear as ClearIcon } from "@mui/icons-material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import FormControl from "@mui/material/FormControl";
+import Modal from "@mui/material/Modal";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import React, { useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+
+export default function RejectButton({
+  height = 40,
+  width = 100,
+}: {
+  height?: number;
+  width?: number;
+}) {
```

Comment:
You actually don't need this; if you specify the default values like you did on lines 14-15, Typescript can figure it out